scripts-2.0/third_party/pipedrive.js

```diff
@@ -2,14 +2,14 @@
 
 const SERVICE = 'pipedrive';
 
-tcbutton.render('.actionsContent:not(.tc)', {observe: true}, function (elem) {
-    let description = $('.descriptionHead h1 a', elem).textContent;
+tcbutton.render('[data-testid="header-title"]:not(.tc)', {observe: true}, function (elem) {
```

Comment:
Pleas don't use `data-testid` it's not code for testing. Don't we already have access to the `actionsContent` class?